packages/fxa-payments-server/src/components/PlanDetails/index.stories.tsx

```diff
@@ -56,12 +56,7 @@ storiesOf('components/PlanDetail', module)
     </MockApp>
   ))
   .add('localized to xx-pirate', () => (
-    <MockApp
-      appContextValue={{
-        ...defaultAppContext,
-        navigatorLanguages: ['xx-pirate'],
-      }}
-    >
+    <MockApp languages={['xx-pirate']}>
```

Comment:
I just noticed the localization story was broken, because `MockApp` (a Storybook construct) doesn't rely on `navigatorLanguages` prop in `AppContext`. I changed it in the obvious places; here and in the `TermsAndPrivacy` component story.

One alternative is of course to modify `MockApp` to rely on `AppContext` for this, but the `languages` prop approach was an easier change. If people feel strongly one way or the other, happy to take a different path here.